pokete.py

```diff
@@ -444,7 +444,7 @@ def add_poke(self, poke, idx=None):
 
     def give_item(self, item, amount=1):
         """Gives an item to the player"""
-        assert amount > 0, "Amounts have to be positive"
+        assert amount > 0, "Amounts have to be positive."
```

Comment:
Personal thing, but if this is to be changed, I would use an exclamation mark, as the program will "crash" if the assertion fails.

```suggestion
        assert amount > 0, "Amounts have to be positive!"
```